test/installer/scripts/omsadmin_systest.rb

```diff
@@ -296,4 +310,45 @@ def test_reconstruct_from_partial_empty_omsadmin_conf
                  "tmp directory was mistakenly created")
   end
 
+  def test_show_omsagent_proc_limit_default
+    set_omsagent_proc_limit()
+    lso = show_omsagent_proc_limit()
+    assert_match(/^-?\d+$/, lso, "The command to show what should be the default proc limit did not yield an integer.")
+  end
+
+  def test_show_omsagent_proc_limit_5000
+    set_omsagent_proc_limit(5000)
+    lso = show_omsagent_proc_limit()
+    nostr = lso.chomp
+    assert_equal("5000", nostr, "Should have been 5000 process limit.")
+  end
+
+  def test_show_omsagent_proc_limit_two_entries
+    FileUtils.cp("#{@omsadmin_test_dir}/limits-two-settings.conf", @proc_limits_conf)
+    lso = show_omsagent_proc_limit()
+    linearray = lso.split("\n")
+    l = linearray.length
+    m = linearray[0].chomp
+    v = linearray[l-1].chomp
```

Comment:
nit: these single-char variables could be named more descriptively; reading the code at the moment does not immediately reveal what their purpose is.